ci_feature/Dockerfile

```diff
@@ -5,6 +5,7 @@ LABEL vendor=Microsoft\ Corp \
     com.microsoft.version="6.0.0-1"
 ENV tmpdir /opt
 ENV APPLICATIONINSIGHTS_AUTH OTQzNWI0M2YtOTdkNS00ZGVkLThkOTAtYjA0Nzk1OGU2ZTg3
+ENV APPLICATIONINSIGHTS_ENDPOINT https://dc.services.visualstudio.com/v2/track
```

Comment:
why do we need this default end point ? this is already hard coded in ai sdk. in the future when we take a new sdk drop if default endpoint has changed there we need to change here as well.

Reply:
> why do we need this default end point ? this is already hard coded in ai sdk. in the future when we take a new sdk drop if default endpoint has changed there we need to change here as well.
Are you saying, we can use the APPLICATIONINSIGHTS_ENDPOINT only if its not nil otherwise use the default endpoint in the sdk?  yes, we can do that too.

Reply:
btw, AI team changes the default endpoint then that will be bigger breaking change for all AI customers who are older on SDK.  Because of the same reason, I believe, they are  using old dns name "*.visualstudio.com.*".

Reply:
Defaults are managed by SDK and we should specify only overrides. So if APPLICATIONINSIGHTS_ENDPOINT is set we should use it, if not let the SDK handle the default (whatever it may be).